NewHorizons/Handlers/PlanetCreationHandler.cs

```diff
@@ -691,6 +691,11 @@ private static GameObject SharedGenerateBody(NewHorizonsBody body, GameObject go
                 atmosphere = AtmosphereBuilder.Make(go, sector, body.Config.Atmosphere, surfaceSize).GetComponentInChildren<LODGroup>();
             }
 
+            if (body.Config.EyeOfTheUniverse != null)
```

Comment:
Doesn't seem to be anything stopping this being called from outside the eye scene, so we should have that. When validating configs we should log errors if this is not null and the system isnt set to EyeOfTheUniverse